campaigns/management/commands/populatecampaigndata.py

```
@@ -0,0 +1,327 @@
+                on_chain_id=campaign_id,
+                defaults=campaign_defaults
+            )
+
+            action = "Created" if created else "Updated"
+            self.stdout.write(f"  {action} campaign: {campaign.on_chain_id}")
+
+            # Fetch USD prices for the campaign
+            try:
+                campaign.fetch_usd_prices()
+                self.stdout.write(f"  Fetched USD prices for campaign {campaign.on_chain_id}")
+            except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)